cmd/apiserver-watcher/run.go

```diff
@@ -149,9 +164,39 @@ func newHandler(uri *url.URL) (*handler, error) {
 		vips: addrs,
 	}
 	glog.Infof("Using VIPs %v", h.vips)
+
+	if err := os.MkdirAll(filepath.Dir(lockpath), 0755); err != nil {
+		return nil, fmt.Errorf("could not create run directory %s: %w", filepath.Dir(lockpath), err)
+	}
```

Comment:
how was this created before?

Reply:
It was being created by the iptables scripts.